examples/v0/account_management/get_accessible_customers.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example lists the resource names for the customer accounts that the
+authenticating user has access to.
+"""
+
+from __future__ import absolute_import
+
+import argparse
+import six
+import sys
+
+import google.ads.google_ads.client
+
+
+def main(client):
+    customer_service = client.get_service('CustomerService')
+
+    try:
+        accessible_customers = customer_service.list_accessible_customers()
+        resource_names = accessible_customers.resource_names[:200]
```

Comment:
Instead of using a literal here, make the limit an argument that defaults to 200. That way, a user who really wants thousands of customers back can get them without modifying the example.

Reply:
Good idea